web/index.html

```diff
@@ -47,6 +47,26 @@
         ::-ms-reveal {
             display: none;
         }
+        @media screen and (min-width: 768px) {
```

Comment:
Can we move this stuff to `web/index.css`?

Reply:
Is index.css inlined?